src/app/components/homepage-components/navbar/input-field/input-field.component.ts

```diff
@@ -1,17 +1,33 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { MatSelectChange } from '@angular/material/select';
+import { DropdownSelect } from '../../../../interfaces/dropdown-select';
+import { FiltersService } from '../../../../services/all-products/filters/filters.service';
 
 @Component({
   selector: 'app-input-field',
   templateUrl: './input-field.component.html',
-  styleUrls: ['./input-field.component.css']
+  styleUrls: ['./input-field.component.scss'],
+  encapsulation: ViewEncapsulation.None
 })
 
 export class InputFieldComponent implements OnInit {
 
-  constructor() { }
+  public foods: DropdownSelect[] = [
+    { value: 'All categories', viewValue: 'All categories' },
+    { value: 'Fruits', viewValue: 'Fruits' },
+    { value: 'Vegetables', viewValue: 'Vegetables' },
+    { value: 'Berries', viewValue: 'Berries' },
+    { value: 'Nuts', viewValue: 'Nuts' }
```

Comment:
You can also try making these values as Enum

Reply:
+refactored